### ISSUE TITLE
"Discard changes" does not delete new added files/folder

### ISSUE THREAD

**Issue by Thane5** [ForkIssues/TrackerWin#955]
When adding new folders or files to a local repository, you can revert those like any change using the "discard" command, but it will not remove them. This is very irritating since they only don't appear in the "changes" window after reverting, yet they will still show up in windows explorer. 

i was expecting a behavior as in Git Extensions, where you have a checkbox to toggle this behavior in the revert dialog window.

**Comment by DanPristupov** [comment]
> When adding new folders or files to a local repository, you can revert those like any change using the "discard" command, but it will not remove them. 

Do you mean it will not remove the files or the folders?

**Comment by Thane5** [comment]
Exactly, it only hides them in the Fork client, but the file aren't actually deleted.

**Comment by DanPristupov** [comment]
Do those files still have content after you discard them?

**Comment by Thane5** [comment]
My bad, i just tested it again and it seems the files are gone, but the folders are still there.
